client/src/app/services/headerInterceptor.ts

```diff
@@ -0,0 +1,27 @@
+import {
+    HttpEvent,
+    HttpInterceptor,
+    HttpHandler,
+    HttpRequest,
+} from '@angular/common/http';
+import { User } from '../models/user';
+import { AuthenticationService } from './authentication.service'
+
+import { Observable } from 'rxjs';
+import { Injectable } from '@angular/core';
+@Injectable() 
+export class AddHeaderInterceptor implements HttpInterceptor {
```

Comment:
please name it as JwtInterceptor, just to be more clear

Reply:
Changed